docker-compose.yml

```
@@ -1,12 +1,17 @@
     restart: unless-stopped
     tty: true
     environment:
-      - ORIGIN=http://localhost:3000 # No trailing slash, read more below
-      - BACKEND_URL=http://127.0.0.1:8080 # No trailing slash, read more below
+      - ORIGIN=http://localhost:3000 # (Optional) No trailing slash, read more below
       - TZ=America/New_York
-    ports:
-      - '3000:3000'
+    expose:
+      - 3000
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)